vortex-datafusion/src/datatype.rs

```diff
@@ -25,7 +25,7 @@ use vortex_dtype::{DType, Nullability, PType};
 ///
 /// This function will panic if the provided `dtype` is not a StructDType, or if the struct DType
 /// has top-level nullability.
-pub(crate) fn infer_schema(dtype: &DType) -> Schema {
```

Comment:
this can stay `pub(crate)`

Reply:
same as below basically - both needed for benchmarks and a generally useful APi.